README.md

```diff
@@ -234,33 +231,27 @@ An example of a two server deployment is shown below.  The first server holds th
       node.data: true
       node.master: false
       bootstrap.memory_lock: false
-    es_scripts: false
     es_templates: false
     es_version_lock: false
-    ansible_user: ansible
     es_api_port: 9200
     es_plugins:
       - plugin: ingest-attachment
-    
-- hosts: data_nodes
+
+- hosts: data_node_2
   roles:
     - role: elastic.elasticsearch
   vars:
-    es_instance_name: "node2"
-    es_api_port: 9201
+    ansible_user: ansible
     es_config:
       discovery.seed_hosts: "elastic02:9300"
-      http.port: 9201
-      transport.port: 9301
+      http.port: 9200
```

Comment:
These are the defaults right? We can just remove them since before they needed to be overriden to allow multiple instances on the same host.

Reply:
We should left `http.port` as it's a recommendation we have here: 
https://github.com/elastic/ansible-elasticsearch/blob/master/tasks/elasticsearch-parameters.yml#L11-L13 (see also https://github.com/elastic/ansible-elasticsearch/pull/558#discussion_r281584111).

But is it still a valid recommendation?

Reply:
created https://github.com/elastic/ansible-elasticsearch/issues/571 to follow up